extension/chrome/elements/pgp_block.ts

```diff
@@ -436,6 +453,28 @@ Catch.try(async () => {
     }
   };
 
+  const addSecondPart = (message: string, isHtml: boolean = false) => {
+    const pgpBlock = $("#pgp_block")
+      .append('<div id="action_open_full" class="three_dots"><img src="/img/svgs/three-dots.svg" /></div>'); // xss-safe-factory
+    if (isHtml) {
+      pgpBlock.append(`<div id="forwarded_msg" class="forwarded_msg" style="display:none">` + // xss-safe-factory
```

Comment:
Also, it may not need both `id="forwarded_msg"` and `class="forwarded_msg"` unless there is a particular reason for it.

If you want, you could also move `style="display:none"` to css. That will shorten the code considerably, possibly fitting on one line.